index.html

```diff
@@ -37,9 +43,235 @@ <h1>Donice Ubaru</h1>
+          <h4>Project name goes here</h4>
+          <div class="project-skill">
+            <ul>
+              <li>HTML/CSS</li>
+              <li>Ruby on Rails</li>
+              <li>Javascript</li>
+            </ul>
+          </div>
+          <button class="project-button">
+            See this project
+            <span class="material-symbols-outlined">
+              arrow_right_alt
+            </span>
+          </button>
+        </div>
+      </div>
+
+      <div>
+        <img src="./images/project-image-4.svg" alt="project-image-1">
+        <div class="project-details">
+          <h4>Project name goes here</h4>
+          <div class="project-skill">
+            <ul>
+              <li>HTML/CSS</li>
+              <li>Ruby on Rails</li>
+              <li>Javascript</li>
+            </ul>
+          </div>
+          <button class="project-button">
+            See this project
+            <span class="material-symbols-outlined">
+              arrow_right_alt
+            </span>
+          </button>
+        </div>
+      </div>
+
+      <div>
+        <img src="./images/project-image-5.svg" alt="project-image-1">
+        <div class="project-details">
+          <h4>Project name goes here</h4>
+          <div class="project-skill">
+            <ul>
+              <li>HTML/CSS</li>
+              <li>Ruby on Rails</li>
+              <li>Javascript</li>
+            </ul>
+          </div>
+          <button class="project-button">
+            See this project
+            <span class="material-symbols-outlined">
+              arrow_right_alt
+            </span>
+          </button>
+        </div>
+      </div>
+    </div>
+  </section>
+
+  <section>
+
+    <h2>About</h2>
+    <div id="About-myself">
+      <p>
+        I can help you build a product , feature or website Look through some of my work and experience! If you like
+        what
+        you see and have a project you need coded, don’t hesitate to contact me.
+      </p>
+      <button type="button">
+        Get my CV
+        <span class="material-symbols-outlined">download</span>
+      </button>
+
+      <div class="skill">
+        <div class="skill-card" id="language">
+          <h4>Languages</h4>
+          <ul>
+            <li>Javascript(ES5/ES6)</li>
+            <li>Ruby</li>
+            <li>HTML</li>
+            <li>CSS</li>
+          </ul>
+        </div>
+
+        <div class="skill-card" id="frameworks">
+          <h4>Frameworks</h4>
+          <ul>
+            <li>React.js</li>
+            <li>Ruby on rails</li>
+            <li>RSpec</li>
+            <li>Capibara</li>
+          </ul>
+        </div>
+
+        <div class="skill-card" id="skills">
+          <h4>Skills</h4>
+          <ul>
+            <li>Database Management</li>
+            <li>CLI</li>
+            <li>API Design</li>
+            <li>Version Control</li>
+            <li>Web Development</li>
+          </ul>
+        </div>
```

Comment:
- [ ] Kindly fix the font-weight and likewise the top and bottom padding for each of these cards to ensure you have a pixel-perfect design.

- Your design
<img width="137" alt="image" src="https://github.com/donice/Portfolio-Microverse/assets/41163571/c7217d12-e786-4312-9d54-c8702e6a4758">

- Figma design
<img width="158" alt="image" src="https://github.com/donice/Portfolio-Microverse/assets/41163571/925e3cb2-8d84-4735-8b12-b5a2cbb5a81e">